Avo.js

```diff
@@ -170,13 +169,13 @@
   };
 
   InternalAvoLogger = {
-    logEventSent: function logEventSent(eventName, eventProperties, userProperties) {
-      const message = "Event Sent:" + eventName + "Event Props:" + JSON.stringify(eventProperties) + "User Props:" + JSON.stringify(userProperties);
+    logEventSent: function logEventSent(eventName, eventProperties, userProperties, groupProperties) {
+      const message = "Event Sent:" + eventName + "Event Props:" + JSON.stringify(eventProperties) + "User Props:" + JSON.stringify(userProperties) + "Group Props:" + JSON.stringify(groupProperties);
 
       if (__AVO_LOGGER__ && __AVO_LOGGER__.logDebug && __AVO_LOGGER__.logDebug(__AVO_ENV__, message)) {
         return
       }
-      typeof console !== 'undefined' && console.log("[avo] Event Sent:", eventName, "Event Props:", eventProperties, "User Props:", userProperties);
+      typeof console !== 'undefined' && console.log("[avo] Event Sent:", eventName, "Event Props:", eventProperties, "User Props:", userProperties, "Group Props:", groupProperties);
```

Comment:
why are group properties coming in here now, something that was forgotten?

Reply:
We haven't pulled Avo in this project since we launched Group properties 🚀